livekit-plugins/livekit-plugins-openai/livekit/plugins/openai/llm.py

```diff
@@ -27,24 +43,26 @@ def __init__(
         *,
         model: str | ChatModels = "gpt-4o",
         client: openai.AsyncClient | None = None,
+        image_detail: ImageDetail = "low",
```

Comment:
This shouldn't in the LLM constructor but in the ChatImage 
See above (I added a `detail` field)

Reply:
I'm not sure if this is LLM specific or not. Although I do recognize that images ought to be able to have different sizing. Maybe ChatImage can simply have an LLM-agnostic width x height and OAI llm maps that to the nearest `low | medium | high`?